api/opencl_runtime_layer.asciidoc

```diff
@@ -5875,10 +5875,10 @@ include::{generated}/api/protos/clEnqueueReleaseExternalMemObjectsKHR.txt[]
     command to complete.
 
 Applications must release the memory objects that are acquired using
-{clEnqueueReleaseExternalMemObjectsKHR} before using them through any
-commands in the other API.
-This is to guarantee that the state of memory objects is up-to-date and they
-are accessible to the other API.
+{clEnqueueReleaseExternalMemObjectsKHR} before using them through any commands
+outside of OpenCL.
```

Comment:
Possible additional improvements:

1. Disambiguate "them", because the memory objects do not have any meaning outside of OpenCL.  Should this be referring to the external memory handles used to create the memory objects instead?  Should it be the memory that those handles represent?
2. Disambiguate "any commands".  Should this be "functions" instead?  What is the right term?